source/bin/tvm.rs

```diff
@@ -154,10 +166,8 @@ mod tests {
     #[test]
     fn case_usage() {
         assert_eq!(
-            "Usage: tvm [options] [file]\n".to_string() +
-            "Options:\n" +
-            "    -v, --version    Print version.\n" +
-            "    -h, --help       This help.",
+            "Usage: tvm [options] [file]\nOptions:\n    -v, --version    \
```

Comment:
Testing the result of a function against it's exact body is not what I would use as a unit test. This reduces quality of tests, and per your definition in the readme, the quality of the whole codebase.